content/sensu-go/5.17/reference/secrets.md

```diff
@@ -0,0 +1,218 @@
+-------------|------
+description  | Top-level attribute that specifies the resource type. For secrets configuration, the type should always be `Secret`.
+required     | Required for secrets configuration in `wrapped-json` or `yaml` format.
+type         | String
+example      | {{< highlight shell >}}"type": "Secret"{{< /highlight >}}
+
+api_version  | 
+-------------|------
+description  | Top-level attribute that specifies the Sensu API group and version. For secrets configuration in this version of Sensu, the `api_version` should always be `secrets/v1`.
+required     | Required for secrets configuration in `wrapped-json` or `yaml` format.
+type         | String
+example      | {{< highlight shell >}}"api_version": "secrets/v1"{{< /highlight >}}
+
+metadata     |      |
+-------------|------
+description  | Top-level scope that contains the secret's `name` and `namespace`.
+required     | true
+type         | Map of key-value pairs
+example      | {{< highlight shell >}}
+"metadata": {
+  "name": "sensu-ansible-token",
+  "namespace": "default"
+}
+{{< /highlight >}}
+
+spec         | 
+-------------|------
+description  | Top-level map that includes secrets configuration [spec attributes][8].
+required     | Required for secrets configuration in `wrapped-json` or `yaml` format.
+type         | Map of key-value pairs
+example      | {{< highlight shell >}}
+"spec": {
+  "id": "ANSIBLE_TOKEN",
+  "provider": "env"
+}
+{{< /highlight >}}
+
+### Metadata attributes
+
+name         |      |
+-------------|------
+description  | The name for the secret that is used internally by Sensu.
+required     | true
+type         | String
+example      | {{< highlight shell >}}"name": "sensu-ansible-token"{{< /highlight >}}
+
+namespace    |      |
+-------------|------
+description  | [Sensu RBAC namespace][9] that the secret belongs to.
+required     | true
+type         | String
+example      | {{< highlight shell >}}"namespace": "default"{{< /highlight >}}
+
+### Spec attributes
+
+id           | 
+-------------|------ 
+description  | The identifying key for the provider to retrieve the secret.
+required     | true
+type         | String
+example      | {{< highlight shell >}}"id": "secret/ansible#token"{{< /highlight >}}
+
+provider     | 
+-------------|------ 
+description  | The name of the Sensu provider with the secret.
+required     | true
+type         | String
+example      | {{< highlight shell >}}"provider": "ansible_vault"{{< /highlight >}}
+
+## Secret configuration
+
+Use the [Secrets API][2] and [sensuctl][3] to create, view, and manage your secrets configuration.
+To manage secrets configuration with sensuctl, configure sensuctl as the default [`admin` user][6].
+
+The [standard sensuctl subcommands][4] are available for secrets (list, info, and delete).
+
+To list all secrets:
+
+{{< highlight shell >}}
+sensuctl secret list
+{{< /highlight >}}
+
+To see a secret's status:
+
+{{< highlight shell >}}
+sensuctl secret info SECRET_NAME
```

Comment:
Is this SECRET_NAME the metadata.name from the Secret setup?

Reply:
Yup!

Reply:
Yes